docs/virtual_machines/instancetypes.md

```diff
@@ -309,6 +309,123 @@ $ kubectl get vmi/vm-cirros-csmall -o json | jq .spec.domain.cpu
 }
 ```
 
+## expand-spec
+
+A VM subresource API and `virtctl` subcommand are provided to allow users to obtain a copy of their existing or yet to be defined VM with any referenced instance types or preferences expanded within the core spec of the VM. 
```

Comment:
```suggestion
The VM `expand-spec` subresource API and `virtctl expand` subcommand allow users to view an existing or yet-to-be defined VM with any referenced instance types or preferences expanded within the core spec of the VM. 
```
It took me a few reads to understand this so my suggestion may have changed the meaning if I've misunderstood.